sktime/registry/_scitype.py

```diff
@@ -7,6 +7,15 @@
 from sktime.registry._base_classes import get_base_class_register
 
 
+def is_a_scitype(obj, desired_scitype):
+    scitypes = scitype(
+        obj, force_single_scitype=False, coerce_to_list=True, raise_on_unknown=False
+    )
+    if isinstance(desired_scitype, str):
+        desired_scitype = [desired_scitype]
+    return any([scitype in desired_scitype for scitype in scitypes])
```

Comment:
minor nitpick, is it clearer if you would be using a set intersection, and check that it is non-empty?

Since the lists are all very small, I don't think there is much efficiency to gain, but it still might be a gain, even.

Reply:
Done